episodes/05-writing-scripts.md

```diff
@@ -330,16 +330,12 @@ using a transfer program, it needs to be installed on your local machine, not yo
 
 ## Transferring Data Between your Local Machine and the Cloud
 
-If you're using Windows with PuTTY instead of Git Bash, please select the alternative option here:
-<select id="id_platform" name="platformlist" onchange="change_content_by_platform('id_platform');return false;">
+If you are using Linux, Mac OS, or Windows with Git Bash on your local machine, you can use
```

Comment:
```suggestion
If you're using Linux, Mac OS, or Windows with Git Bash on your local machine, you can use
```
Thanks @vhmcck, I didn't realise I had changed `you're` to `you are` - and consistency is good so I've changed it back